dev/com.ibm.websphere.appserver.features/visibility/public/pages-3.0/io.openliberty.pages-3.0.feature

```diff
@@ -29,10 +29,10 @@ IBM-API-Package: jakarta.servlet.jsp;  type="spec", \
  org.apache.el;  type="internal", \
  org.apache.el.lang;  type="internal", \
  org.apache.jasper.el;  type="internal"
-IBM-ShortName: jsp-3.0
+IBM-ShortName: pages-3.0
 IBM-SPI-Package: com.ibm.wsspi.jsp.taglib.config
```

Comment:
Thinking you should also put a WLP-AlsoKnownAs: jsp-3.0

Reply:
actually I have an issue opened to do that: https://github.com/OpenLiberty/open-liberty/issues/14317

I'll work that after I get this initial merge done.